src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs

```diff
@@ -62,6 +62,7 @@ private IEnumerable<LinkResource> BuildLinks(PurchaseOrder model, IEnumerable<st
                 if (this.authService.HasPermissionFor(AuthorisedAction.PurchaseOrderUpdate, privileges))
                 {
                     yield return new LinkResource { Rel = "edit", Href = this.GetLocation(model) };
+                    yield return new LinkResource { Rel = "over-book", Href = this.GetLocation(model) };
```

Comment:
looks better. href should match the url in the module i.e. $"{this.GetLocation(model)}/over-book"